src/inject/GPMInject/interface/mini.js

```diff
@@ -16,7 +16,7 @@ window.wait(() => {
     Emitter.fire('mini', { state: true });
     oldSize = remote.getCurrentWindow().getSize();
     mainWindow.setSize(MINI_SIZE, MINI_SIZE);
-    mainWindow.setMaximumSize(MINI_SIZE, MINI_SIZE);
```

Comment:
I'm not sure removing this is the best idea.  Some users will end up resizing to be 400,500 or even 600x600 and then it will look awful because the album art is only 300x300